PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchView.swift

```diff
@@ -4,17 +4,101 @@ import Textile
 struct SearchView: View {
     @ObservedObject
     var viewModel: SearchViewModel
+    var showResultsView: Bool = false
 
     var body: some View {
-        if viewModel.showRecentSearches == true, !viewModel.recentSearches.isEmpty {
-            RecentSearchView(recentSearches: viewModel.recentSearches)
-        } else if let emptyState = viewModel.emptyState {
-            EmptyStateView(viewModel: emptyState)
-                .padding(Margins.normal.rawValue)
+        if showResultsView {
+            ResultsView(viewModel: viewModel)
+        } else {
+            if viewModel.showRecentSearches == true, !viewModel.recentSearches.isEmpty {
+                RecentSearchView(recentSearches: viewModel.recentSearches)
+            } else if let emptyState = viewModel.emptyState {
+                EmptyStateView(viewModel: emptyState)
+                    .padding(Margins.normal.rawValue)
+            }
         }
     }
 }
 
+struct ResultsView: View {
+    @ObservedObject
+    var viewModel: SearchViewModel
+
+    var listItemViewModel: ItemsListItemCell.Model = ItemsListItemCell.Model(
```

Comment:
```suggestion
    let listItemViewModel: ItemsListItemCell.Model = ItemsListItemCell.Model(
```

If these aren't being mutated, we can use `let`.

Reply:
updated!